.github/workflows/deploy_images.yml

```diff
@@ -101,6 +101,11 @@ jobs:
         with:
           java-version: 11
           distribution: temurin
+      - name: Set up Python 3.10.9
+        id: python
```

Comment:
oh, it was a bad idea to rebase. This code was removed in master

Reply:
I guess the problem is in `force-pushed the master branch from d92be76 to 65c52ab`

Reply:
rebased current PR